prow/cmd/build/controller.go

```diff
@@ -57,6 +58,20 @@ const (
 	controllerName = "prow-build-crd"
 )
 
+type pjClient struct {
+	pjc       prowjobset.Interface
```

Comment:
Why not resolve to the correct namespace at load time?

Reply:
Fixed by using the `ProwJobInterface` instead.